---
title: Dual-indexed library preparation of gDNA for Illumina sequencing
author: James S. Santangelo
date: __Last updated\:__ \today
bibliography: references.bib
toc: true
toc-title: Table of contents
numbersections: true
linkcolor: blue
---

# Version history

#### v1.1.1 (Latest)

- Minor typo fixes
- Added details on performin 2nd PCR raction to increase concentrations
- Added details on pooling into equimolar pools for sequencing

#### v1.1.0

- Added colored links
- Minor typo and syntax changes
- Updated end-repair master mix
- Added post shearing bead cleanup
- Autogenerated references
- Added YAML header to generate table of contents, title, author, and last updated

#### v1.0.0 (July 10, 2019)

- Initial commit of protocol

\pagebreak

# Setting up

## Reagents

**Table 1:** Reagents and consumables used during library prep, including supplier and catalog numbers. All costs are approximate.

| Reagent | Supplier | Catalog # | Cost (CAD) | Cost/reaction |
|---------|----------|-----------|------------|---------------|
| Bioruptor tubes | Diagenode | C30010011 | 263.26 | 0.53 |
| PCR Tubes | VWR | 93001-118 | 86.10 | 2.87 |
| Pipette tips 200 uL| TR-222-C-L | Corning | 41.54 | 0.83 |
| Pipetter tips 10 uL | T-300-L | Corning | 38.75 | 0.77 |
| dNTP | Fisher | Fermentas R1121 | 265 | 0.04 |
| ATP | Fisher | Fermentas R0441 | 52.25 | 0.07 |
| Tango buffer | Fisher | Fermentas BY5 | 14.3 | 0.01 |
| T4 Polynucleotide Kinase | Fisher | EK0032 | 249 | 0.17 |
| T4 DNA polymerase | Fisher | EP0062 | 249 | 0.33 |
| Nuclease free water | Fisher | NA | 33.2 | 0.02 |
| dATP | Fisher | LS10216018 | 61.41  | 0.04 |
| Taq | Fisher | LS18038067 | 276.22 | 0.55 |
| iTrusR2-stubRCp  | IDT | NA | 65.4 | 0.01 |
| iTrusR1-stub | IDT | NA | 35.4 | 0.003 |
| T4 DNA ligase  | Fisher | Fermentas EL0011 | 279.67 | 0.28 |
| Sera-Mag SpeedBeads | Fisher | 09-981-123 | 686 | 0.23 |
| PEG-8000 | VWR | 97061-098 | 64.83  | 0.02 |
| EDTA | VWR | 97062-656 | 61.14  | 0.02 |
| Tris Buffer | VWR | 97062-974 | 70.82 | 0.02 |
| Tween 20 | VWR | 97061-384 | 64.23  | 0.02 |
| NaCl | VWR | CA12001-384 | 56.34  | 0.02 |
| 50 bp DNA ladder | Fisher | FERSM0371 | 63.75 | 0.08 |
| ITru5_Group1_8-Indices | IDT | NA | 198.64 | 0.59 |
| ITru7_Group1_12-Indices | IDT | NA | 255.56 | 0.82 |
| Phusion HiFi polymerase | NEB | M0530S | 116.19 | 0.29 |
| **Total** | NA | NA | NA | **8.60** |

## Lab equipment

1. PCR thermocycler with heated lid
2. Gel rigs
3. 96-well plate magnet (e.g., Dynamag-96 well #12331D)
4. Racks for 0.2 mL PCR tubes or plates
5. PCR plates or tubes
6. 10 and 100 &#956;L multichannel or singlechannel pipettes

## Before starting {#sec:2.3}

This protocol is used for preparing dual-indexed, Illumina-compatible gDNA libraries for whole genome sequencing. It doesn't rely on any commercial kits; rather, the protocol uses home-made master mixes for all reactions to minimize costs. References will be provided throughout to credit the original publications.

Briefly, this protocol uses an "on-bead" library preparation (@Fisher2011) with "home-brewed" master mixes (@Kobs1997; @Meyer2010; @Fisher2011) and SPRI beads (@Rohland2012), and incorporates i5 and i7 Ilummina indices into adapters through PCR (@Glenn2019).

This protocol assumes that you have:

1. Hydrated all of index-containing forward and reverse PCR primers (described in supplementary file: _Protocol for preparing 1.25 nmol iTru primer aliquots for use during library preparation_ in @Glenn2019). These are diluted to 5 &#956;M.
2. Annealed the adapter stubs (i.e., iTrusR2-stubRCp, iTrusR1-stub) to form the Y-yoke adapter (supplementary file:_Protocol for preparing double-stranded iTru adapters for use during library preparation_ in @Glenn2019). These are diluted to 5 &#956;M. This is referred to as the _adaptor mix_ throughout the protocol_
3. Prepared the SPRI bead solution, which replaces Ampure XP beads for cleaning enzymatic reations (@Rohland2012; reproduced in supplementary file:_Protocol for preparing an inexpensive substitute for AMPure_ in @Glenn2019). Be sure to also prepare a PEG solution lacking the actual SPRI beads (described in the same file, which will be used throughout the protocol.
4. Prepared a fresh aliquot of 80% ethanol. Addition of 0.05% Tween-20 to this PEG/NaCl solution will greatly facilitate the bead washed

# Protocol

## DNA Shearing and Cleanup

### Shearing

1. Normalize DNA to 10 ng/&#956;L (quantified by Qubit) in 25 &#956;L of low-TE (10 mM Tris-HCl ph8 and 0.1 mM EDTA) in Bioruptor tubes. You can shear more than 25 &#956;L if you want multiple shots at the library prep per smaple.
    * _Note:_ We've had success using lower concentrations if DNA quantity is an issue
2. Shear DNA by sonication to mean fragment size of \~500 bp.
    * For our samples we used 3 cycles of 15 seconds ON, 30 seconds OFF on a Bioruptor Pico
    * _Note:_ We sheared DNA using a Bioruptor Pico, but any acoustic shearing device would likely be fine (e.g., Covaris).
    * _Note:_ It's a good idea to perform a few different rounds of shearing with different settings and running these on a gel and/or Bioanalyzer make sure you're getting the correct fragment size distribution.
3. After shearing, move 25 &#956;L of DNA to empty PCR tube and proceed to post-shearing cleanup.

### Post-shearing cleanup {#sec:3.1.2}

1. Add 20 &#956;L of SPRI bead solution to each sample. This is a 0.8X Bead:DNA ratio, which is meant to remove fragment < \~250 base pairs.
2. Incubate at room temperature for 5 minutes.
3. Place samples on magnetic rack and allow the beads to bind to the side of the tube. The supernatant should be clear.
4. Remove and discard the supernatant (the DNA is bound to the beads so won't be lost).
5. Add 80 &#956;L of freshly prepared 80% ethanol. Let sit on magnetic rack for 30 to 60 seconds and discard ethanol using a pipette.
6. Perform another ethanol wash as in step 5.
7. Remove any residual ethanol using a 10 &#956;L a pipette. Let samples sit uncovered at room temperature for 5 to 10 minutes or until all ethanol has evaporated.
    * **Important:** Do not let beads dry out too much as this will reduce yield. Beads should be matte brown (not glossy). Dry beads with begin to crack.
8. Elute DNA in 20 &#956;L low-TE (or dH<sup>2</sup>0). Pipette mix to homogenize beads and TE. The DNA will now be in solution.

**Note:** The beads from the above reaction will be kept through all subsequent enzymatic reactions and reused for all SPRI bead cleanups (except PCR). The beads do not interfere with the reactions (@Fisher2011).

## End-repair and cleanup

### End-repair

1. Prepare the end-repair master mix below shown in table 2. Each reaction requires 10 &#956;L. It's a good idea to prepare more than you'll need (e.g., enough for 9 samples if you're preparing 8 libraries)

    **Table 2:** End-repair master mix from @Meyer2010. All volumes in &#956;L

    | Reagent      | Units | Initial conc. | Vol.   | Final conc. |
    |--------------|-------|---------------|--------|-------------|
    | Tango buffer | X     | 10            | 3      | 1           |
    | ATP          | mM    | 10            | 3      | 1           |
    | dNTP         | mM    | 10            | 0.3    | 0.1         |
    | T4 PNK       | U/uL  | 10            | 1.5    | 0.5         |
    | T4 POL       | U/uL  | 5             | 0.6    | 0.1         |
    | Water        | NA    | NA            | 1.6    | NA          |
    | **Total**    | NA    | NA            | **10** | NA          |

2. Pipette 10 &#956;L of master mix in to 25 &#956;L of each DNA sample. Mix throughly by pipetting or flicking.
3. Incubate reation in thermocycler using the following conditions (from Fisher _et al._ 2011):
    * 15 mins at 12°C followed by 15 mins at 25°C (30 mins total).

### Post end-repair cleanup

1. Add 84 &#956;L of 20% PEG-8000/2.5 mM NaCl (i.e. SPRI bead solution without the beads) to samples. This is a 2.8X Bead:DNA ratio (@Fisher2011).
2. Perform bead cleanup as in [@sec:3.1.2], eluting DNA in 20 &#956;L of low-TE.


## A-tailing and cleanup

### A-tailing

1. Prepare the A-tailing master mix shown in table 3. Each sample requires 10 &#956;L.

    **Table 3:** See @Kobs1997 and [this wiki from the Rausher lab at Duke](https://wiki.duke.edu/display/rausherlab/A-tailing+of+PCR+products) for details on A-tailing. All volumes in &#956;L

    | Reagent    | Units | Initial conc. | Vol.  | Final conc. |
    |------------|-------|---------------|-------|-------------|
    | Taq buffer | X     | 10            | 3     | 1           |
    | dATP       | mM    | 10            | 0.6   | 0.2         |
    | Taq        | U     | 5             | 3     | 0.5         |
    | MgCl2      | mM    | 50            | 1.5   | 2.5         |
    | Water      | NA    | NA            | 1.9   | NA          |
    | **Total**  | NA    | NA            | **10**| NA          |

2. Add 10 &#956;L of master mix to 20 &#956;L each DNA sample.
3. Incubate in thermocycler for 30 minutes at 70°C.

### Post A-tailing cleanup

1. Add 66 &#956;L of 20% PEG-8000/2.5 mM NaCl (i.e. SPRI bead solution without the beads) to samples. This is a 2.2X Bead:DNA ratio (@Fisher2011).
2. Perform bead cleanup as in [@sec:3.1.2], eluting DNA in 20 &#956;L of low-TE.

## Adapter ligation and cleanup

### Adapter ligation

1. Prepare the adapter ligation master mix shown in table 4. Each sample requires 10 &#956;L.

    **Table 4:** Adapter ligation master mix from @Meyer2010. All volumes in &#956;L

    | Reagent       | Units | Initial conc. | Vol.  | Final conc. |
    |---------------|-------|---------------|-------|-------------|
    | Ligase buffer | X     | 10            | 3.5   | 1           |
    | PEG-4000      | mM    | 50            | 3.5   | 5           |
    | T4 Ligase     | U     | 5             | 0.88  | 0.125       |
    | Water         | NA    | NA            | 2.12  | NA          |
    | **Total**     | NA    | NA            | **10**| NA          |

2. Add 5 &#956;L of 5 &#956;M adapter mix (see point 2 in [@sec:2.3]) to each DNA sample.
3. Add 10 &#956;L of master mix to each sample. Pipette mix thoroughly. The total volume should now be 35 &#956;L.
4. Incubate in thermocycler for 30 minutes at 25°C

### Post-adapter ligation cleanup

1. Add 31.5 &#956;L of 20% PEG/2.5 mM NaCl (i.e. speedbeads without the beads) to samples. This is a 0.9X bead:DNA ratio. This will get rid of smaller DNA fragments (e.g., < \~250 bp) like adapter dimers that may have formed during the ligation reaction.
2. Perform bead cleanup as in [@sec:3.1.2], eluting DNA in 20 &#956;L of low-TE.
3. After elution, place beads on magnet and carefully transfer the supernatant to new tube/plate by slowly drawing up the supernatant with a pipette. The DNA is in the supernatant. This will be used for indexing PCR.

## Indexing PCR and cleanup

### Indexing PCR

1. Prepare the PCR master mix shown in table 5. Each reaction requires 12.5 &#956;L.

    **Table 5:** [PCR master mix](https://international.neb.com/Protocols/0001/01/01/pcr-protocol-m0530). All volumes in &#956;L

    | Reagent            | Units | Initial conc.| Vol.     | Final conc. |
    |--------------------|-------|--------------|----------|-------------|
    | Phusion buffer     | X     | 5            | 5        | 1           |
    | dNTP               | mM    | 10           | 0.5      | 0.2         |
    | Phusion polymerase | U/uL  | 2            | 0.25     | 0.02        |
    | Water              | NA    | NA           | 6.75     | NA          |
    | **Total**          | NA    | NA           | **12.5** | NA          |

2. Add 12.5 &#956;L of master mix to empty PCR tubes/plates corresponding to the number of samples to be amplified and indexed.
3. Add 10 &#956;L of DNA sample to each well containing PCR master mix.
4. Add 1.25 &#956;L of forward primer and 1.25 &#956;L of reverse primer to each sample.
    * _Note:_ Each sample should receive a unique combination of forward and reverse primers since these will incorporate the indices into the libraries. 8 forward + 12 reverse primers can uniquely tag 96 individuals.
5. Perform PCR using the following conditions. See @Glenn2019 for details on these primers.
    * 98 &#176;C for 30 seconds
    * 8 to 14 cycles of:
        * 98 &#176;C for 30 seconds
        * 65 &#176;C for 30 seconds
        * 72 &#176;C for 60 seconds
    * 72 &#176;C for 1 minute

### 2nd indexing PCR and pooling (optional)

Because only 10 &#956;L of the pre-PCR (i.e., ligated) library is used in the indexing PCR, each library prep allows for a total of two PCR reactions. If you find that final library concentrations (i.e., after cleaning, see [@sec:3.5.3]) are too low, you can perform a second PCR reaction on the remaining pre-PCR library using the same conditions as above. These two reaction can then be pooled prior to cleaning, and then eluted in a lower volume to increase concentrations (see [@sec:3.5.3] below).

### Post PCR cleanup {#sec:3.5.3}

1. Add 1X the post-PCR DNA volume in SPRI bead solution to each sample. If you only performed a single PCR reaction, this should be approximately 25 &#956;L. If you pooled two 25 &#956;L PCR reactions, this should be approximately 50 &#956;L of SPRI bead solution. This should remvove primer dimers and other small fragments remaining from PCR.
2. Perform bead cleanup as in [@sec:3.1.2], eluting in 23 &#956;L (i.e., 20 &#956;L for sequencing and 3 &#956;L for post-cleaning QC)
3. QC library by visualizing on a gel using the 50 bp Fermentas ladder or by passing through an Agilent Bioanalyzer with a DNA 1000 high sensitivity chip.

## Pooling for sequencing

For multiplexed sequencing, dual-indexed libraries need to be pooled into equimolar ratio pools to ensure approximately equal sequencing coverage. This is done in two steps: (1) Creating equimolar dilutions for each sample, and (2) Creating an equimolar pool for all samples to be multiplexed on a single lane. 

### Creating equimolar dilutions

1. Create equimolar dilutions for each sample, diluting the concentration down to that of the least concentrated sample. For highly concentrated samples, serial dilution may be required. The script in step 1 will output the volume required for each library, in addition to the volume of low-TE required to create the equimolar pool. It will additionally indicate whether serial dilutions are necessary and any required volumes. 
    * _Note:_ [This script](https://github.com/ness-lab/illumina-pooling) can facilitate determining the library volumes required to create equimolar pools for sequencing.

### Creating equimolar pool

1. Pipette the same volume from each sample (which are now normalized to the same concentration from the step above) into the appropriate tube. You should have 1 tube for each sequencing lane.
    * _Note:_ To avoid pipetting error associated with using very small volumes, it's best to ensure that at least 2 &#956;L is being taken from each sample when creating the equimolar pool. 

## Testing the libraries

We tested the protocol above by shotgun sequencing two white clover (_Trifolium repens_) genomes to 35X coverage on a lane of HiSeq 4000. [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) reports for both sequencing libraries are available [HERE](./fastqc_reports/). While the reports show some warnings, these can be largely eliminated by the adapter trimming performed using [bbduk](https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbduk-guide/). The adapter contamination occured because we sheared DNA to a mean size of 300 bp, resulting adapter read-though of some smaller fragments during sequencing. We now recommend shaering to 500bp to minimize this. Nonetheless, few reads were lost and all remaining reads were high quality.

# References









